dash-pipeline/dockerfiles/Dockerfile.saichallenger-client-bldr

```diff
@@ -10,16 +10,18 @@ ADD SAI/SAI/test/ptf /SAI/test/ptf
 # Install PTF test framework & test-cases from SAI repo
 ADD SAI/SAI/ptf /SAI/ptf/
 
-# Install dependencies for dpugen==0.0.5
-RUN apt-get update && \
-    apt-get install -y python3-lxml
+# Install dependencies for dpugen==0.2.3
+RUN apt-get update -y
```

Comment:
does apt-get update need -y?

Reply:
@r12f  its not compulsory, added -y incase if we get prompts while running apt-get so that it run non-interactively